v3/tasks/20250313-v3-quantamm/input.ts

```diff
@@ -0,0 +1,153 @@
+import { MONTH } from '@helpers/time';
+import { Task, TaskMode } from '@src';
+import { ethers } from 'ethers';
+import { bn } from '@helpers/numbers';
+import { ZERO_ADDRESS, ZERO_BYTES32 } from '@helpers/constants';
+
+export type QuantAMMDeploymentInputParams = {
+  Vault: string;
+  PauseWindowDuration: number;
+  USDC: string;
+  WBTC: string;
+  FactoryVersion: string;
+  PoolVersion: string;
+};
+
+const Vault = new Task('20241204-v3-vault', TaskMode.READ_ONLY);
+const USDC = new Task('00000000-tokens', TaskMode.READ_ONLY);
+const WBTC = new Task('00000000-tokens', TaskMode.READ_ONLY);
+
+const ChainlinkSepoliaDataFeedETH = "0x694AA1769357215DE4FAC081bf1f309aDC325306";
+const ChainlinkSepoliaDataFeedUSDC = "0xA2F78ab2355fe2f984D808B5CeE7FD0A93D5270E";
+const ChainlinkSepoliaDataFeedBTC = "0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43";
+
+const BaseVersion = { version: 1, deployment: '20250312-v3-quantamm' };
+
+export default {
+  Vault,
+  PauseWindowDuration: 4 * 12 * MONTH,
+  USDC,
+  WBTC,
+  FactoryVersion: JSON.stringify({ name: 'QuantAMMWeightedPool', ...BaseVersion }),
+  PoolVersion: JSON.stringify({ name: 'QuantAMMWeightedPool', ...BaseVersion }),
+  ChainlinkSepoliaDataFeedETH,
+  ChainlinkSepoliaDataFeedUSDC,
+  ChainlinkSepoliaDataFeedBTC,
```

Comment:
If you have a set of ChainlinkDataFeed<Token> addresses for each chain, you can just export those as 3 variables, and the framework will set it to the correct address depending on the "--network" argument:

sepolia: {
    ChainlinkDataFeedETH: "0x694AA1769357215DE4FAC081bf1f309aDC325306",
    ChainlinkDataFeedUSDC: "0xA2F78ab2355fe2f984D808B5CeE7FD0A93D5270E",
    ChainlinkDataFeedBTC: "0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43",
}, ... other chains